<launch>
  <arg name="image_topic"/> <!-- TODO: use remapping instead -->
	<node pkg="zbar_detector" type="barcode_detector" name="barcode_detector" output="screen">
	  <param name="image_topic" type="string" value="$(arg image_topic)" />
    <param name="show_cv_window" type="bool" value="true" />
	</node>
</launch>
